### PR TITLE
add lesson tables for new lessons [WIP]

### PR DESCRIPTION
**Pull request by ErinBecker**

### CODE REVIEW

File: pages/lessons.md [hunk 1]
```diff
@@ -22,11 +22,11 @@ Data Carpentry workshops are domain-specific, so that we are teaching researcher
 
 ## Curriculum materials under development or consideration
 
-- [Digital humanities curriculum](#dh-curriculum)
```
Comment (ErinBecker) [inline pages/lessons.md]: Re-ordered these to reflect relative "done-ness" of curricula.

File: pages/lessons.md [hunk 2]
```diff
@@ -383,23 +383,38 @@ offerings. If you are interested in being involved in developing one of these le
...
 model, see our [Curriculum Development Handbook](https://cdh.carpentries.org) for information about how to get started.
 
-### <a id="dh-curriculum"></a> Digital Humanities Curriculum
-
-Many groups are piloting different versions of this curriculum. There is not yet one set of lessons under active development.
-
-If you are interested in following or being involved in development of this curriculum, please sign up for the [dh-curriculum email list](https://carpentries.topicbox.com/groups/curriculum-dh)
-
-### <a id="image-analysis-curriculum"></a> Image analysis Curriculum
+### <a id="image-processing-curriculum"></a> Image Processing Curriculum
```
Comment (ErinBecker) [inline pages/lessons.md]: Update name to "Image Processing" from "Image analysis"

**Comment by ErinBecker** [comment]
@maneesha  - could you please preview this locally to make sure the tables show up properly before merging? Thanks so much!

**Comment by maneesha** [comment]
Sorry I didn't get to take a look before it got merged.  Tables look good!

**Pull request merged**